src/commands/dev/dev.js

```diff
@@ -312,6 +323,46 @@ const startPollingForAPIAuthentication = async function (options) {
   await helper(siteInfo)
 }
 
+/**
+ * @param {boolean|string} edgeInspect
+ * @param {boolean|string} edgeInspectBrk
+ * @returns {InspectSettings}
+ */
+const generateInspectSettings = (edgeInspect, edgeInspectBrk) => {
+  const enabled = Boolean(edgeInspect) || Boolean(edgeInspectBrk)
+  const pause = Boolean(edgeInspectBrk)
+  const getAddress = () => {
+    if (edgeInspect) {
+      return typeof edgeInspect === 'string' ? edgeInspect : undefined
+    }
+    if (edgeInspectBrk) {
+      return typeof edgeInspectBrk === 'string' ? edgeInspectBrk : undefined
+    }
+  }
+
+  return {
+    enabled,
+    pause,
+    address: getAddress(),
+  }
+}
+
+const validateShortFlagArgs = (args) => {
+  if (args.includes('=')) {
```

Comment:
[dust] Not a big deal, but the particular case of people using  short form flag with a `=` would only happen if the arg _starts_ with a `=`, not when it exists elsewhere in the arg?

```suggestion
  if (args.startsWith('=')) {
```